packages/@react-stately/layout/src/ListLayout.ts

```diff
@@ -25,7 +25,8 @@ export type ListLayoutOptions<T> = {
   indentationForItem?: (collection: Collection<Node<T>>, key: Key) => number,
   collator?: Intl.Collator,
   loaderHeight?: number,
-  placeholderHeight?: number
+  placeholderHeight?: number,
+  allowDisabledKeyFocus?: boolean
```

Comment:
New option for ListLayout, makes disabled rows focusable via keyboard navigation, defaults to false.